assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -1,18 +1,57 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Link } from "react-router-dom";
+import { useQuery, useMutation } from "@apollo/react-hooks";
 import useIsEditing from "../../../hooks/useIsEditing";
-import { VISIBILITY_OPTIONS } from "../../../services/global-vars";
+import { toastWrapper } from "../../../services/helpers";
+import {
+  VISIBILITY_OPTIONS,
+  RIGHTS_STATEMENTS,
+  PRESERVATION_LEVELS
+} from "../../../services/global-vars";
+import { GET_COLLECTIONS } from "../../Collection/collection.query";
+import { UPDATE_WORK, ADD_WORK_TO_COLLECTION, GET_WORK } from "../work.query";
 import { useForm } from "react-hook-form";
+import UITagNotYetSupported from "../../UI/TagNotYetSupported";
 
 const WorkTabsAdministrative = ({ work }) => {
   const [isEditing, setIsEditing] = useIsEditing();
   const { register, handleSubmit } = useForm();
+  const [updateWork] = useMutation(UPDATE_WORK, {
+    onCompleted({ updateWork }) {
+      setIsEditing(false);
+    }
+  });
+
+  const [addWorkToCollection] = useMutation(ADD_WORK_TO_COLLECTION, {
```

Comment:
Can you add a "TODO: // comment here" reminding us to take another look at this, and describing why we're leaving this code block in this commit?